base/clortho-get

```diff
@@ -12,12 +12,11 @@ if ENV['AWS_REGION'].nil?
 end
 
 if File.file?('/etc/aws/credentials')
-  YAML.load_file('/etc/aws/credentials') do |creds|
-    Aws.config[:credentials] = Aws::Credentials.new(
-      creds['aws_access_key_id'],
-      creds['aws_secret_access_key']
-    )
-  end
+  creds = YAML.load_file('/etc/aws/credentials')
```

Comment:
It's a shame that `YAML#load_file` doesn't work that way. That's a pretty common idiom in the `File` class.